bitcoin/build.rs

```diff
@@ -1,4 +1,26 @@
-const MSRV_MINOR: u64 = 63;
+use std::fs;
+
+pub fn parse_msrv_minor() -> u64 {
+    let f = "Cargo.toml";
+    let key = "rust-version";
+
+    let cargo_toml = fs::read_to_string(f).expect("manifest exists");
+
```

Comment:
`f` and the local var `cargo_toml` are entangled now so this is worse than not defining `f`.

Reply:
Also `f` and `key` are entangled. I'd just leave the strings inline and be done with it.

Reply:
By "entangled" I mean the value in `key` is dependent on the value in `f`.

Reply:
ah oops, sorry about that.  I'll fixit.